### ISSUE TITLE
[1.18.2] The menu does not open

### ISSUE THREAD

**Issue by antirain1** [ThinkingStudios/Litematica-Forge#9]
hello again!
Your last fix really helped, now this mod is client-side!
but now a new problem has arisen, the Litematica menu does not open.
I tried all the buttons - nothing, and and in the settings of the keys in minecraft there is no item related to Litematica....

**Comment by googledino** [comment]
I had the same problem but when I downgraded to 0.1.4 (the previous 1.18.2 version) it is working and on servers as well.

**Comment by antirain1** [comment]
> I had the same problem but when I downgraded to 0.1.4 (the previous 1.18.2 version) it is working and on servers as well.

could you write which name version of .jar of MaLiLib and forgematica you use, and it works on the client-side?

**Comment by googledino** [comment]
forgematica-0.1.4-mc1.18.2
MaFgLib-0.1.5-mc1.18.2

Yes it works on the client-side

**Comment by antirain1** [comment]
> forgematica-0.1.4-mc1.18.2

Are you sure that forgematica version-0.1.4-mc1.18.2 supports client-side capability? 
I just tested it, it does not let me into the server, this problem was solved in the Forgematica version-0.1.5-mc1.18.2, but the menu does not work there ¯\_(ツ)_/¯

**Comment by googledino** [comment]
I am playing on a modded server so I'm guessing that has something to do with it, but the server does not have the mod installed. I'm using forge 40.2.4 if it helps

**Comment by antirain1** [comment]
> forgematica version-0.1.4-mc1.18.2



> I am playing on a modded server so I'm guessing that has something to do with it, but the server does not have the mod installed. I'm using forge 40.2.4 if it helps

yes, i guess apparently the forge version affects this, I now have version 40.2.9.

**Comment by antirain1** [comment]
So, the bottom line for us and for the creator of this is it's time:
**MaFgLib-0.1.5-mc1.18.2** + 
**Forgematica-0.1.5-mc1.18.2**
client side = ✅
menu open = ❌

**MaFgLib-0.1.5-mc1.18.2** + 
**forgematica version-0.1.4-mc1.18.2**
client side = ❌
menu open = ✅

we will wait for a response from the port developer ¯_(ツ)_/¯

**Comment by gentlewatcher** [comment]
please update and push this also for 1.19.2 same issues and version numbers, .15 and .15 also does not open the menu.

**Comment by TexBlock** [comment]
Ok, this is mafglib issue, i will used forge event in mafglib next version
